src/Initializer/typedefs.hpp

```diff
@@ -457,4 +457,11 @@ struct MemoryProperties {
   size_t pagesizeStack{PAGESIZE_STACK};
 };
 
+struct TravellingWaveParameters {
+  std::array<double, 3> origin;
```

Comment:
uninformed question:
why double and not real?

Reply:
`InitialField::evaluate` takes `points` as double, so I made the origin also double....
https://github.com/SeisSol/SeisSol/blob/d08b8eb4171e938514d35a9ca32c2756e16f3af4/src/Physics/InitialField.h#L17

Reply:
ok!